api/sheets/index.md

```diff
@@ -8,38 +8,58 @@ has_children: true
 
-Install the add-on from [G Suite Marketplace](https://gsuite.google.com/marketplace/app/data_commons/454343067575). To enable the add-on in a document, click "Add-ons > Data Commons > Fill place dcids". You may use the resulting sidebar to start [finding dcids](/api/sheets/get_dcid.html) in the United States, or close it and reopen it at any time. Note that none of the custom functions will work in a given document until you have enabled the add-on by choosing "Fill place dcids".
+> **NOTE:**
+>
+> None of the custom Data Commons Sheets functions (described below) will work in a given document until you have enabled the add-on by opening the **Fill place dcids** sidebar.
 
-![](/assets/images/sheets/sheets_menu_bar.png)
+## Finding a Place’s DCID
 
-## Finding a place's DCID
+Data Commons’ Sheets add-on provides the ability to look up a location’s [DCID](/glossary.html#dcid), within the application by using the **Fill place dcids** feature. To use this feature to find a place’s DCID:
 
-Data Commons' Sheets add-on ships with the ability to look up a location's [DCID](/glossary.html), or unique Data Commons identifier, within the application. To use this feature, select the final destination cell of the place [DCID](/glossary.html). Go to the Sheets menu bar, and click:
+1. Select the destination cell where you want to add a place’s DCID.
+2. Click in the search box for the **Fill place dcids for selected cells** section on the right side of the page to start finding DCIDs in the United States.
+3. Start typing the name of the place you are searching for. From the drop-down menu, select the place you want, and its DCID appears in the cell that you selected. For example, the following image shows the place names that match “Hawaii”.
+   ![Google Sheets search box](/assets/images/sheets/sheets_search_box.png)
 
-**"Add-ons > Data Commons > Fill place dcids"**
+Note that this feature currently only supports places in the United States.
 
-![](/assets/images/sheets/sheets_menu_bar.png)
+## Introduction to the Sheets API Custom Functions
 
-A sidebar will appear on the right of the sheet, with a search bar where you can start typing the name of the place you want. From the drop down menu, select the place you want, and its [DCID](/glossary.html) will appear in the cell that you selected.
+The Data Commons Sheets API includes the five formulas listed in the following table. Click the links in the table for detailed information on each formula.
 
-![](/assets/images/sheets/sheets_search_box.png)
+| **Formula**                                                                                  | **Description**                           |
+|----------------------------------------------------------------------------------------------|-------------------------------------------|
+| [=DCGETNAME(dcids)](/api/sheets/get_name.html)                 | Returns the names associated with a DCID. |
```

Comment:
Use "Retrieves" instead of "Returns" throughout?

Reply:
good call - done